src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,76 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+
 public class WorkWithFile {
+    public static final String DATA_LINE_SEPARATOR = ",";
+    public static final String DATA_BUILDER_SEPARATOR = " ";
+    public static final int OPERATION_POSITION = 0;
+    public static final int AMOUNT_POSITION = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        readFromFile(file, toFileName);
+    }
+
+    private void readFromFile(File file, String toFileName) {
+        try {
+            StringBuilder builder = new StringBuilder();
+            BufferedReader reader = new BufferedReader(new FileReader(file));
+            String value = reader.readLine();
+            while (value != null) {
+                builder.append(value)
+                        .append(" ");
+                value = reader.readLine();
+            }
+            writeToFile(calculateData(builder), toFileName);
```

Comment:
Method called `readFromFile` must read. It can not write

Reply:
I don't fully understand this request. After all, the read data must be sent for further calculation.
I'll make a `return `in the method.